examples/atum-examples/src/main/scala/za/co/absa/atum/examples/SampleMeasurements1.scala

```diff
@@ -56,7 +59,11 @@ object SampleMeasurements1 extends Eventually {
       .parquet("data/output/stage1_job_results")
 
     eventually(timeout(scaled(10.seconds)), interval(scaled(500.millis))) {
+<<<<<<< HEAD:examples/atum-examples/src/main/scala/za/co/absa/atum/examples/SampleMeasurements1.scala
       if (!Files.exists(Paths.get("data/output/stage1_job_results/_INFO"))) {
+=======
+      if (!fs.exists(new Path("data/output/stage1_job_results/_INFO"))) {
+>>>>>>> master:examples/src/main/scala/za/co/absa/atum/examples/SampleMeasurements1.scala
```

Comment:
Needs resolving, IMHO.

**This leads me to the conclusion that the examples tests are, indeed, not run!**

Reply:
Should be done now